karma.conf.js

```diff
@@ -86,7 +86,7 @@ module.exports = function (config) {
       "/app/": appAssets
     },
 
-    exclude: [],
+    exclude: ['node_modules/**/*spec.js', 'node_modules/**/*spec.ts'],
```

Comment:
Where were these files included by Karma? Why is the additional exclusion necessary?

Reply:
Without excluding tests in node_modules karma tries to run tests in packages and fails with the following error:
`     "stack": "(SystemJS) XHR error (404 Not Found) loading http://localhost:9877typescript\n\tError loading http://localhost:9877typescript as \"typescript\" from http://localhost:9877node_modules/@angular/tsc-wrapped/test/collector.spec.js",
    "originalErr": {
       "__zone_symbol__currentTask": {
         "type": "microTask",
         "state": "notScheduled",
         "source": "Promise.then",
         "zone": "<root>",
...`